src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -55,6 +56,7 @@ type ReduxProps = {
   layout: Layout;
   node?: NodeParamsType;
   refreshInterval: PollIntervalInMs;
+  mtlsEnabled: boolean;
```

Comment:
So enabling mesh-wide mtls means all namespaces, right?

Reply:
yep, is for all namespaces. Should I rename this variable? Like `meshMtlsEnabled`?

Reply:
no, it's fine as is.